app/assets/stylesheets/markdown-editor.scss

```diff
@@ -0,0 +1,39 @@
+.md-editor {
+  .md-footer,
+  .md-header {
+    background: $sidebars-background;
+    display: block;
+    padding: 6px 4px;
+  }
+
+  .md-header {
+    height: 42px;
+    border-bottom: 1px solid $border-grey;
+    margin: 0;
+  }
+
+  .md-preview {
+    background: $white;
+    min-height: 90px;
+    overflow: auto;
+    z-index: 100;
+    position: relative;
+    padding: 6px 12px;
+    color: $text-color;
+    height: auto !important;
+    width: 100% !important;
+  }
+
+  .md-controls {
+    float: right;
+    padding: 3px;
+
+    .md-control {
+      color: $text-grey;
+      padding: 3px 3px 3px 10px;
+      right: 5px;
+    }
+  }
+}
+
+#publisher_textarea_wrapper:not(.active) .md-header { display: none; }
```

Comment:
Selector `publisher_textarea_wrapper` should be written in lowercase with hyphens